### ISSUE TITLE
Extended public key support

### ISSUE THREAD

**Issue by armas-mk** [iotaledger/firefly#1384]
## Description

Generate a public key for a wallet that can be used to query nodes to show the available balance in an aggregated output.

Same as xPub key in many popular cryptocurrencies.

## Motivation

It would make it easier to integrate IOTA wallets with tracker apps (eg.: cointracker.io). I'm sure there are plenty of other use cases as well.

## Requirements

Write a list of what you want this feature to do.

1. Generate external public key or a master view public key

## Open questions (optional)

I assume it will also require some changes into the node software's API. Shall i create a FEATURE request there as well with a link to this issue?

## Are you planning to do it yourself in a pull request?

No. Sorry, I don't have the time.

**Comment by cvarley100** [comment]
Unfortunately, I don't think xPub is available for our signature scheme (cc @Wollac).

**Comment by Wollac** [comment]
Yes, xPub is defined for Bitcoin's bip-32 but not slip-10 with Ed25519 which is e.g. used in IOTA.

There are some ideas how to implement a similar scheme, but nothing is standardized or vetted enough yet.
We will of course revisit this once this has changed.
